### ISSUE TITLE
Double clicking on file in Tree view to open permanent tab not working if containing folder is a symlink

### ISSUE THREAD

**Issue by solbjorn** [atom/tabs#173]
Despite on commit 73658a93, closed #171 and string "Double-clicking a file in the Tree View now makes it a permanent tab when using preview tabs" in Atom 0.208.0 changelog, this feature is not working. Double-clicking in Tree view makes nothing, while single-click open preview tab for file. Tested also with emptied .atom config folder and with --safe parameter.

Atom 0.208.0 (with tabs 0.77) built from sources, Arch Linux x64.

**Comment by mnquintana** [comment]
I can't reproduce this. What is the output of `atom.getVersion()` in the console?

**Comment by solbjorn** [comment]
"0.208.0". Double-click on tab opens it permanently, but doulbe-click on file in Tree view does not.

**Comment by mnquintana** [comment]
@solbjorn What is the output of atom.getVersion() in the console?

**Comment by mnquintana** [comment]
Can you paste your `config.cson` here?

**Comment by solbjorn** [comment]
Of course:
```coffeescript
"*":
  "exception-reporting":
    userId: "480b20de-e736-263e-a240-4205e8c90e87"
  welcome:
    showOnStartup: false
  core:
    themes: [
      "one-dark-ui"
      "solarized-dark-syntax"
    ]
  editor:
    fontFamily: "'M+ 1mn regular'"
    invisibles:
      {}
  minimap:
    plugins:
      "git-diff": true
      bookmarks: true
      selection: true
      "color-highlight": true
      "find-and-replace": true
  "one-dark-ui":
    {}
  "atom-beautify":
    _analyticsUserId: "13adc827-7ceb-4c02-bdb7-7a6d1b173567"
    c_configPath: "/usr/share/uncrustify/defaults.cfg"
    cpp_configPath: "/usr/share/uncrustify/defaults.cfg"
    cs_configPath: "/usr/share/uncrustify/defaults.cfg"
  "atom-color-highlight":
    {}
  "autocomplete-plus":
    {}
  "git-diff":
    {}
  tabs:
    usePreviewTabs: true
  "spell-check":
    {}
  "open-recent":
    {}
  "git-projects":
    rootPath: "~/Документы/Программы;"
```

**Comment by mwolfindale** [comment]
+1 on Ubuntu 15.04, Atom version 0.207.0

**Comment by mnquintana** [comment]
@mwolfindale This was added in 0.208.0, not 0.207.0 - you need to upgrade to 0.208.0 first.

**Comment by mwolfindale** [comment]
Ah sorry! OK, I'll upgrade and let you know if any issues still. Thanks :)

**Comment by kevinsawicki** [comment]
Tried it out on Ubuntu 14.04 and I was unable to reproduce

**Comment by solbjorn** [comment]
Same on prebuilt 0.208.0 (deb package) with empited .atom folder and --safe parameter.

**Comment by kevinsawicki** [comment]
@solbjorn are you double clicking on the the name of the file or the area to the left/right of the text?

**Comment by solbjorn** [comment]
I tried double-clicking everywhere. I can't reproduce this bug on elementary OS 0.3 with Atom installed from deb package, but on Arch this happens with both built and deb-packaged Atom. Clearing .atom, .cache/Atom, .config/Atom folders does not help.

**Comment by solbjorn** [comment]
IMPORTANT: double-clicking on files in Tree View does not work ONLY if root folder has been accessed through symlink! That's why we can't reproduce this on elementary and Ubuntu.

**Comment by mwolfindale** [comment]
Reproduced this on Ubuntu 15.04, Atom 0.208.0, double clicking on files in tree view opens permanent tab __except__ when the containing folder is a symlink (this can either be the root project folder, _or_ any folder in the project)

**Comment by mnquintana** [comment]
Thanks for the updates everyone! I can reproduce this with Atom 0.209.0 and OS X 10.10.3 for files inside of symlinked directories.

**Comment by benogle** [comment]
This is fixed on master and the 1.5.x line which is in beta now. See https://github.com/atom/atom/issues/9165 for more information.
